scripts/aliases/utilities.mjs

```diff
@@ -0,0 +1,9 @@
+export const aliasStore = new Map([
+	['filterNullAndUndefined', 'filterNullish'],
+	['filterNullAndUndefinedAndEmpty', 'filterNullishOrEmpty'],
+	['filterNullAndUndefinedAndZero', 'filterNullishOrZero'],
+	['isNullOrUndefined', 'isNullish'],
+	['isNullOrUndefinedOrEmpty', 'isNullishOrEmpty'],
+	['isNullOrUndefinedOrZero', 'isNullishOrZero'],
+	['sleep', 'sleepSync']
```

Comment:
`sleep` and `sleepSync` aren't the same, they should be split.

Reply:
The aliases are to indicate they target the same file in the mapping, not that the functions are actual aliases.